test/src/pages/details/details-tab.test.js

```diff
@@ -1,4 +1,4 @@
-import DetailsTab from '~/pages/details/details-tab/details-tab';
+import DetailsTab from '~/pages/details/details-tab/details-tab.jsx';
 
```

Comment:
hm, why do you need to specify the extension?  Does the resolve/extensions need to have `.jsx` added to the web pack config? 

Seems like that might be brittle if type of files change to `js`

Reply:
It's complicated, and has to do with the somewhat byzantine build system. I'm not sure what's possible or not.

Reply:
Oh yeah, I forgot that you don't use web pack.  Ignore that then, sorry!